src/main/java/br/com/excaladashboard/models/Cliente.java

```diff
@@ -16,20 +14,76 @@ public class Cliente {
     @Column(name = "id")
     private Long id;
 
+    @Column(name = "ativo", length = 256)
+    private String ativo;
+
     @Column(name = "empresa", length = 256)
     private String empresa;
 
-    @Column(name = "responsavel", length = 256)
-    private String responsavel;
+    @Column(name = "telefone", length = 256)
+    private String telefone;
+
+    @Column(name = "email", length = 256)
+    private String email;
+
+    @Column(name = "cnpj", length = 256)
+    private String cnpj;
+
+    @Column(name = "cpf", length = 256)
+    private String cpf;
+
+    @Column(name = "pagamento_em_dia", length = 256)
+    private String pagamentoDia;
+
+    @Column(name = "contrato_assinado", length = 256)
+    private String contratoAssinado;
+
+    @Column(name = "preco_fixo_contrato", precision = 15, scale = 2)
+    private Double precoFixoContrato;
+
+    @Column(name = "taxa_variavel_contrato", precision = 15, scale = 2)
+    private Double taxaVariavel;
+
+    @Column(name = "inicio_contrato")
+    @Temporal(TemporalType.DATE)
+    private Date inicioContrato;
+
+    @Column(name = "tempo_contrato_ativo")
+    @Temporal(TemporalType.DATE)
+    private Date contratoAtivo;
+
+    @Column(name = "status_contrato", length = 256)
+    private String statusContrato;
+
+    @Column(name = "conta_anuncios", length = 256)
+    private String contaAnuncios;
```

Comment:
Esse campo precisa de relacionamento com a classe Conta.

```suggestion
    usar many to one e one to many, um cliente pode ter varias contas, mas uma conta ser de um cliente.
    private List<Conta> contas;
```